tuning/__main__.py

```diff
@@ -0,0 +1,28 @@
+from tuning.tune import run_tune
```

Comment:
Not sure what this is used for? benchmarking? if so maybe move it to the benchmarking directory?